draft-packages/tag/KaizenDraft/Tag/Tag.tsx

```diff
@@ -122,7 +146,7 @@ const Tag = (props: TagProps) => {
               onMouseDown={onMouseDown}
               onMouseLeave={onMouseLeave}
             >
-              <span className={styles.background} />
+              {/* <span className={styles.background} /> */}
```

Comment:
If this is not needed, we should remove it, rather than leaving it here commented out.